pytoda/smiles/transforms.py

```diff
@@ -29,7 +169,7 @@ def __init__(self, smiles_language: SMILESLanguage) -> None:
         """
         self.smiles_language = smiles_language
 
-    def __call__(self, smiles: str) -> Indexes:
+    def __call__(self, smiles: str) -> Union[Indexes, Tensor]:
```

Comment:
isn't the output always a list of ints, aka what we define as `Indexes` in `types.py`?